flake.nix

```diff
@@ -22,6 +22,7 @@
         # --- packages ----
 
         packages = flake-utils.lib.flattenTree {
+          libsecp256k1 = pkgs.callPackage (import ./nix/libsecp256k1.nix) {};
```

Comment:
`libsecp256k1 = pkgs.secp256k1.overrideAttrs (attrs: { configureFlags = attrs.configureFlags ++ ["--enable-static"]; })`

Reply:
Ah, thanks! Let me try that out :)

Reply:
Done, thanks!